charters/DataPortal/charter.md

```diff
@@ -0,0 +1,73 @@
+# Data Portal
+
+
+## Description
+The Data Portal is the landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides users interested in the HCA with an easy and efficient way to explore and use data at cellular resolution.
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyze, and make available the data in the Human Cell Atlas project.
+
+*Data Browser* A DCP application which provides a faceted search capability.
+
+## Objectives
+* Provide the access point for users to search, contribute to, or learn about, the cellular resolution and other data which is part of the HCA collection, at petabyte scale on public clouds.
+* Work with the other DCP teams to provide a positive user experience for all users of the DCP website.
+* Provide a common look and feel across the entire DCP website.
+* Provide easy-to-use methods for users to identify a subset of the DCP data for download and/or further analysis.
+
+## In-scope
+
+### Core capabilities
+* Implement a visual design which addresses the needs and goals of the identified main user personas. It is driven by the user experience research insights and interaction design guidelines provided by the User Experience (UX) team.
```

Comment:
You shouldn't say `It is driven by the user experience research` but you should state what you mean by It, if you mean the visual design, say the visual design, if you mean the UI, say the UI

Reply:
Changed "It" to "The visual design".